interface/main/tabs/templates/tabs_template.php

```diff
@@ -38,6 +38,12 @@
 
         <!-- ko  foreach: tabs.tabsList -->
         <div class="frameDisplay" data-bind="visible:visible">
+            <div class="col-sm-12" class="loader" data-bind="visible:loader">
+                <div class="text-center">
+                    <i class="fa fa-spinner fa-pulse fa-fw" style="font-size: 140px; color: var(--gray700); padding: 20px"></i>
```

Comment:
Font size here should be using rem instead of pixels to allow responsive fonts. This is how it should be written: `font-size: 8.75rem`. You can get this number by multiplying pixels by 0.0625 to get the rem.

Reply:
@mdsupport Didn't think about that -- we should be using font awesome's size classes here.

Reply:
Thank you for guiding me 😃 

I cant see fa-9x in Font Awesome library (ver : 4.7.0)  
![image](https://user-images.githubusercontent.com/47683556/76058819-59da2d00-5fa3-11ea-9ef8-43fe150fc20f.png)
I included loading text with loader 👍

Reply:
@tywrenn To get fa-9x we need to edit the source and create custom incrementations, right?